__tests__/app.test.js

```diff
@@ -168,4 +181,54 @@ describe('GET /api/articles/:article:id/comments', () => {
              expect(error.msg).toBe('bad request')
          })        
      });
-    })
\ No newline at end of file
+    })
+    describe('POST  /api/articles/:article_id/comments', () => {
+
+        test('should return new object with the values of the new message', () => {
+            
+    
+        const newComment={
+            username:'butter_bridge',
+            body:'blah blah blah,yada,yada ,yada'
+        }
+        return request(app)
+        .post('/api/articles/2/comments')
+        .expect(201)
+        .send(newComment)
+        .then(({body})=>{
+            expect(body.comment).toEqual(
+                {
+                    comment_id: 19,
+                    body: 'blah blah blah,yada,yada ,yada',
+                    article_id: 2,
+                    author: 'butter_bridge',
+                    votes: 0,
+                    created_at: expect.any(String)
+                  })
+
+        })
+    });
+    test('should respon with an error if given a article_id of a valid type that does not exist in our data', () => {
```

Comment:
we could have tests for given username doesnt exist, one for body or username not provided, and one for ignores unnecessary properties on the posted object